src/components/LoginRegister.js

```diff
@@ -0,0 +1,234 @@
+import {
+  VStack,
+  Heading,
+  Button,
+  Box,
+  FormControl,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  Input,
+  FormHelperText,
+  FormLabel,
+} from '@chakra-ui/react';
+import { useState, useRef } from 'react';
+
+const LoginRegister = (props) => {
+  const [showPassword, setShowPassword] = useState(false);
+  const filePicker = useRef(null);
+  const handleShowPasswordChange = () => {
+    setShowPassword(!showPassword);
+  };
+  return (
+    <VStack w="100%" h="100vh" mx="left" bgGradient="linear(to-r, #E8F6EF, #E8F6EF)">
+      {props.activeOption === 'login' ? (
```

Comment:
This is ok for this task but in the next (for authentication), we'll want to use react router to render routes